pwndbg/commands/heap.py

```diff
@@ -1,46 +1,127 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-"""
-Heap commands.
-"""
 from __future__ import print_function
-import argparse
 import gdb
+
+import pwndbg.vmmap
 import pwndbg.commands
+import pwndbg.symbol
+import pwndbg.memory
 
-@pwndbg.commands.OnlyWhenRunning
-@pwndbg.commands.ParsedCommand
-def brk(n=0):
-    '''Get the address of brk(n=0)'''
-    gdb.execute('call brk(%i)' % n)
+from pwndbg.color import bold, yellow, red, underline
+
+PREV_INUSE = 1
+IS_MMAPED = 2
+NON_MAIN_ARENA = 4
+
+def get_main_arena(addr=None):
+    if addr == None:
+        main_arena = gdb.lookup_symbol('main_arena')[0].value()
+    else:
+        if isinstance(addr, (long, int)):
+            addr = hex(addr)
+        main_arena = gdb.parse_and_eval('(struct malloc_state)*' + addr)
```

Comment:
I'd recommend using lookup_type and then creating the object directly if it exists